app-admin/procinfo-ng/files/procinfo-ng-2.0.304-link-tinfo.patch

```diff
@@ -0,0 +1,15 @@
+--- configure.in	2020-02-12 11:51:21.652062146 +0300
++++ configure.in.new	2020-02-12 11:52:33.150686479 +0300
+@@ -39,10 +39,10 @@
+ 
+ if test "$enable_maintainer_mode" = "yes"; then
+ 	CXXFLAGS="-O0 -g3 --pipe -Wall"
+-	LDFLAGS="-lncurses"
++	LDFLAGS="-lncurses -ltinfo"
+ else
+ 	CXXFLAGS="$CXXFLAGS -pipe -Wall"
+-	LDFLAGS="-s -lncurses"
++	LDFLAGS="-s -lncurses -ltinfo"
```

Comment:
please use pkg-config for retrieving the flags

Reply:
done, the file's been renamed procinfo-ng-2.0.304-use-pkgconfig.patch as well